contribs/dvrp/src/main/java/org/matsim/contrib/dvrp/passenger/PassengerRequest.java

```diff
@@ -46,4 +46,6 @@ default double getLatestStartTime() {
 	List<Id<Person>> getPassengerIds();
 
 	String getMode();
+
+	int getPartySize();
```

Comment:
`getPartySize()` may be a confusing to others. IMO `getSize()` or `getPassengerCount()` are more self-descriptive.

If you want to use `getPartySize()`, then I suggest adding a javadoc comment.

Reply:
Thanks, I'll go with PassengerCount